www/tolo00/cv02/view/businessCardBack.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+    require_once __DIR__ . '/../model/Company.php';
+
+?>
+
+<div class="page back-page">
+    <div class="grid-x align-middle">
+        <div class="cell medium-7">
+            <div class="grid-y">
+                <div class="cell">
+                    <h3><?= Company::COMPANY_NAME ?></h3>
+                </div>
+
+                <div class="cell">
+                    <i class="fa-solid fa-person"></i><?=  $person->getFullName() ?>

```

Comment:
jak se pak nekdo dozvi, kde je definovana promenna `$person`?, v `__DIR__ . '/../model/Company.php';` ?

Reply:
Asi máte na mysli soubory `businessCardBack.php` a `businessCardFront.php` ..?

Napadá mě jenom asi něco na způsob:

```php
// This file is loaded in foreach ($persons as $person) loop in "index.php"

/** @var Person $person */
$person = $person ?? null;

if (!$person) {
    return;
}
```

Tzn. dát informaci v komentáři kde se soubor includuje a oanotovat tu proměnnou aby IDE napovídal a pokud by `$person` bylo `null`, tak buď nic nevykreslit, nebo třeba hodit nějakou `\Exception`.

Jestli to takto však stále není dostatečné, tak jsem bohužel bez nápadů a chtěl bych poprosit o radu, ať to mohu dokončit, děkuji